charts/ztka/templates/NOTES.txt

```diff
@@ -2,16 +2,18 @@
 {{- if .Values.ingress.enabled }}
   Open {{ include "ztka.consoleFQDNWithScheme" . }} in browser.
 {{- else if .Values.deploy.contour.enable }}
-  {{- if (eq (index .Values.contour.envoy.service.annotations "service.beta.kubernetes.io/aws-load-balancer-type" | toString ) "nlb") }}
   Get load balancer address via:
   kubectl get service {{ .Release.Name }}-contour-envoy --namespace {{ .Release.Namespace }} -o jsonpath='{.status.loadBalancer.ingress[0].hostname}'
```

Comment:
Do we get the hostname in case of others. Wouldn't it be just an ip?

Reply:
Good point. I think following command will work for both hostname and IP.
```
kubectl get service myztka-contour-envoy -n paralus-systems
NAME                   TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)                      AGE
myztka-contour-envoy   LoadBalancer   10.109.29.31   34.121.xx.xx     80:32000/TCP,443:31450/TCP   7d23h
```
What do you think?

Reply:
Yeah, I think that might be better.